website/docs/backends/types/terraform-enterprise.html.md

```diff
@@ -10,37 +10,43 @@ description: |-
 
 **Kind: Standard (with no locking)**
 
-Stores the state in [Terraform Enterprise](https://www.terraform.io/docs/providers/index.html).
+Reads and writes state from a [Terraform Enterprise](/docs/enterprise/index.hml)
```

Comment:
The Travis-CI run caught that this URL is typoed as `index.hml` instead of `index.html`.

Reply:
good job travis. Will fix!